<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多功能计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <widget class="QWidget" name="polynomialTab">
       <attribute name="title">
        <string>多项式计算</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>多项式 1：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="polynomialInput1">
            <property name="placeholderText">
             <string>示例：3^0+2^1+1^2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>多项式 2：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="polynomialInput2">
            <property name="placeholderText">
             <string>示例：1^0+1^1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>x 值：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="xValueInput">
            <property name="placeholderText">
             <string>请输入整数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>相加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="subtractButton">
            <property name="text">
             <string>相减</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multiplyButton">
            <property name="text">
             <string>相乘</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="derivativeButton">
            <property name="text">
             <string>求导</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="evaluateButton">
            <property name="text">
             <string>计算 P(x)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="polynomialResultLabel">
          <property name="text">
           <string>结果将显示在这里</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="expressionTab">
       <attribute name="title">
        <string>表达式计算器</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>请输入表达式（支持 + - * / ^ 和括号）：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="expressionInput">
          <property name="placeholderText">
           <string>示例：(1+2)*3-4/2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="evaluateExpressionButton">
          <property name="text">
           <string>计算结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="expressionResultLabel">
          <property name="text">
           <string>结果将显示在这里</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
